src/completed.js

```diff
@@ -0,0 +1,51 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export function loadFromStorage() {
+  let savedFormData = localStorage.getItem('tasksList');
+  savedFormData = JSON.parse(savedFormData);
+
+  if (savedFormData) {
+    const tasksArray = savedFormData;
+    const todoList = document.querySelector('.todo-list');
+    todoList.innerHTML = '';
+    tasksArray.forEach((todo) => {
+      todoList.innerHTML += `<li>
+            <div class="taskCont">
+              <input type="checkbox" class="checkBoxesTasks" id="checkbox${todo.index}">
+              <div class="task" id="taskText${todo.index}">${todo.description}</div>
+              <textarea class="textArea" maxlength="255"></textarea>
+            </div>
+            <div class="icon">&#8942</div>
+          </li>
+        `;
+    });
+  }
+}
+
+export function evaluateCheckBox() {
+  const tasksArray = [];
+  const liElements = document.querySelectorAll('li');
+
+  liElements.forEach((liElem, index) => {
+    const labelTask = liElem.querySelector('.task');
+    let labelTaskText;
+    const strike = labelTask.getElementsByTagName('strike');
+    labelTaskText = labelTask.innerHTML;
```

Comment:
- [ ] Please from the screenshot below, you can see the status of the is not updated in the localStorage. Please make sure that the status of the task completion is updated upon user actions. When the checkbox is checked the status should change to `true` and when unchecked the status should change to `false`.

[Screencast from 02-21-2023 11:51:28 PM.webm](https://user-images.githubusercontent.com/92303891/220486407-2df5739e-126c-43ef-8d93-b8fda2ed5958.webm)